config/locales/cy.yml

```diff
@@ -70,9 +70,9 @@ cy:
     devolved_nations:
       applies_to: Yn berthnasol i
       connectors:
-        last_word:
-        two_words:
-      england:
```

Comment:
In the trello card, it says:


> Translation confirmed:
> 
> Standalone, 'England and Wales' is 'Lloegr a Gymru' but if it's got 'i' before it as it does in this context it would be 'Loegr a Gymru'. Also there is a letter missing from the beginning - it should be 'yn' not 'n'. So translation is: 'Yn berthnasol i Loegr a Gymru'.


So should this be `Loegr` rather than `Lloegr`?

Reply:
Oh that's right. I read it the other way around 🤦‍♀️

Reply:
Would you mind updating the screenshot? 

Also , it's England and wales in English, but Wales and England in Welsh. Is that how it works?